CHANGELOG.md

```
@@ -1,5 +1,19 @@
 # Changelog
 
+## [0.21.0](https://github.com/rivenmedia/riven-frontend/compare/v0.20.0...v0.21.0) (2025-05-12)
+
+
+### Features
+
+* add PWA ([437fd52](https://github.com/rivenmedia/riven-frontend/commit/437fd527d87d82a53a2e2365d64081e93e5fa82c))
+* use parse endpoint in manual scraping ([4eb17ff](https://github.com/rivenmedia/riven-frontend/commit/4eb17fff4beac3aea98719b0dd0505731e3e343d))
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)